libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/behandling/Klage.kt

```diff
@@ -391,7 +392,7 @@ data class KlageBlankettPdfgenDTO(
     val ovesendelseTekst: String,
     val klager: String,
     val klageDato: LocalDate,
-) {
+) : PDFMal {
```

Comment:
Hva er tenkt med interfacet, er tanken at det skal implementeres noe fra dette i DTO'ene på sikt?

Reply:
For no er tanken primært å kunne ta inn typa input til PdfGeneratorKlient-en, så vi ikkje måtte sende inn Any eller a | b | c, men ser for meg at vi fint kan utvide med logikk der etter kvart også.